data/Templates/Hl7v2/Resource/_Provenance.liquid

```diff
@@ -3,6 +3,14 @@
     "resource":{
         "resourceType": "Provenance",
         "id":"{{ ID }}",
+        {% if Root_Template -%}
+        {% assign generationInstant = "{{ "" | now }}" -%}
+        {% assign templateVersion = "3.5.1" -%}
```

Comment:
Is it possible to store version in a seperate file like config.json? Seems more friendly to maintain version upgrades.

Reply:
The JSON file can't be directly used by template. We only need to change one variable to maintain the version number either in JSON file or in the Provenance template, and we only change this version number internally, so maybe the effort will be the same. Furthermore, the Provenance resource itself has a meta semantics, so that itself is as special as a "separate version template" and maybe we don't need an extra file with meta semantics.
If the HL7 and C-CDA can use shared templates in the future, I think it's a good idea to use a separate config file.

Reply:
Yes. A seperate file would be better if we have multiple references. Yet I failed to find a solution to load an external variable from config files for templates.